inbm/dispatcher-agent/dispatcher/xmlhandler.py

```diff
@@ -148,7 +148,7 @@ def get_attribute(self, xpath: str, attribute_name: str) -> str:
 
         @param xpath: path to key
         @param attribute_name: name of attribute
-        @return: attribute str if found else None
+        @return: value if found else None
```

Comment:
Clarify the return type in the docstring of `get_attribute` to indicate that it returns a string or raises an exception if the attribute is not found. [medium]